# Joint positions of the robot

Header header

Joints joints